repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/legacy-publication.ftl

```diff
@@ -134,7 +134,8 @@
                 <div class="panel panel--grey push-half--bottom">
                     <h3><@fmt.message key="headers.resources"/></h3>
                     <ul data-uipath="ps.publication.resources">
-                        <#list legacyPublication.attachments as attachment>
+                        ${legacyPublication.getExtAttachments()?size}
+                        <#list legacyPublication.getExtAttachments() as attachment>
                             <li class="attachment">
                                 <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>" onClick="logGoogleAnalyticsEvent('Download attachment','Publication','${attachment.resource.filename}');">${attachment.text}</a>;
```

Comment:
take a look at "external-resource-demo" branch.

```
<@externalstorageLink attachment.resource; url>
    <a title="${attachment.text}" href="${url}" onClick="logGoogleAnalyticsEvent('Download attachment','Publication','${attachment.resource.filename}');">${attachment.text}</a>;
</@externalstorageLink>
```

Reply:
@robValtech could you have a look at this?

Reply:
We'll make the change mentioned above after it is merged to master and has been fully tested etc. Need to progress with this as is for now - blocking our migration